src/interchange/avro.rs

```diff
@@ -612,13 +612,34 @@ impl Encoder {
                         .0;
                     let avro_val = Self::data_to_avro(node.step(&before.schema), &row.unpack())?;
                     // Add wrapper Record with before and after RecordFields
-                    let wrapped_avro_val = Value::Record(vec![
-                        (
-                            "before".into(),
-                            Value::Union(null_idx, Box::from(Value::Null)),
-                        ),
-                        ("after".into(), avro_val),
-                    ]);
+                    let wrapped_avro_val = if let Some(diff) = diff {
+                        if diff >= 0 {
```

Comment:
imo, 
```
while diff >= 0 {
    ...
    diff -= 1;
}

Reply:
I think that is the only meaningful question here -- my understanding on debezium is that the format is really only used for records with primary keys and therefore we have a set and would only ever get +1 / -1 records?